lib/project.rb

```diff
@@ -146,11 +146,23 @@ def check_run_termination
     end
 
     def init_workspace
+      @restored = @options.test.session
```

Comment:
Avoid creating a new state by introducing a new instance variable if possible; you can define a `restored?` method instead.